tests/Avalonia.RenderTests/OpacityMaskTests.cs

```diff
@@ -57,7 +57,7 @@ public async Task Opacity_Mask_Masks_Element()
         }
 
         [Fact]
-        public async Task RenderTansform_Applies_To_Opacity_Mask()
+        public async Task RenderTransform_Applies_To_Opacity_Mask()
```

Comment:
RenderToFile call below has [CallerMemberName] on its parameter. As well as CompareImages call. That's why tests are failing, as expected image files have different/old names.